app/controllers/application_controller.rb

```diff
@@ -1,3 +1,3 @@
-class ApplicationController < ActionController::Base
-  skip_before_action :verify_authenticity_token
+class ApplicationController < ActionController::API
```

Comment:
changing the type of the controller since we are simply an API and not a web applciation